nucypher/policy/conditions/offchain.py

```diff
@@ -0,0 +1,168 @@
+from typing import Any, Optional, Tuple
+
+import requests
+from jsonpath_ng.exceptions import JsonPathLexerError, JsonPathParserError
+from jsonpath_ng.ext import parse
+from marshmallow import fields, post_load, validate
+from marshmallow.fields import Field
+
+from nucypher.policy.conditions.base import AccessControlCondition
+from nucypher.policy.conditions.exceptions import (
+    ConditionEvaluationFailed,
+    InvalidCondition,
+    InvalidConditionLingo,
+)
+from nucypher.policy.conditions.lingo import ConditionType, ReturnValueTest
+from nucypher.policy.conditions.utils import CamelCaseSchema
+from nucypher.utilities.logging import Logger
+
+
+class JSONPathField(Field):
+    default_error_messages = {
+        "invalidType": "Expression of type '{type(value)}' is not valid for JSONPath",
+        "invalid": "'{value}' is not a valid JSONPath expression",
+    }
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        if not isinstance(value, str):
+            raise self.make_error("invalidType", input=value)
```

Comment:
```suggestion
            raise self.make_error("invalidType", value=value)
```
Do you mean `value` here? otherwise, not sure how this format is supposed to work.

Reply:
Oh, I'll take a look at the docs.  Maybe I misunderstood `make_error`.